WcaOnRails/app/javascript/edit-schedule/SchedulesEditor/calendar-utils.js

```diff
@@ -203,13 +210,11 @@ export function singleSelectEvent(event) {
   events.forEach((elem) => {
     if (elem.selected && (event.id != elem.id)) {
       elem.selected = false;
-      // this function might be called while dragging/resizing,
-      // so we'd better remove the class ourselves instead of calling updateEvent!
-      $(".selected-fc-event").removeClass("selected-fc-event");
+      $(scheduleElementSelector).fullCalendar("updateEvent", elem);
     }
   });
   event.selected = true;
-  // We don't render again the element: on dragging/resizing it will be rerendered, else the caller will take care of the update.
+  $(scheduleElementSelector).fullCalendar("updateEvent", event);
   return true;
```

Comment:
Does this function still need to return true/false, or is this unnecessary now?

Reply:
It's not necessary anymore, I just thought that returning information could be better than returning no information :p
I'll remove them and merge this!